Lib/test/test_free_threading/test_frame.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import unittest
+import threading
+
+from test.support import threading_helper
+
+NTHREADS = 6
+
+@threading_helper.requires_working_threading()
+class TestFrame(unittest.TestCase):
+    def test_frame_clear_simultaneous(self):
```

Comment:
This test should be cleaned up:
* gen is not a generator
* the loop in gen does nothing
* foo and frame1 are different names for the same variable